dashboard/streamlit_graphs/sales_over_time.py

```diff
@@ -60,19 +73,24 @@ def fetch_sales_within_date_range(connection, start_date, end_date) -> pd.DataFr
 
 
 def plot_sales_per_hour(connection: extensions.connection,
-                        start_date: datetime, end_date: datetime) -> alt.Chart | None:
+                        start_date: datetime, end_date=None) -> alt.Chart | None:
```

Comment:
We will want the type hint as well as a default value

Reply:
good spot!